apps/atomy-q/API/tests/Unit/Services/AiOperationalAlertPublisherTest.php

```
@@ -0,0 +1,85 @@
+final class AiOperationalAlertPublisherTest extends TestCase
+{
+    public function testPublishSnapshotSendsNotifierAndOutboxAlertsForDegradedCapability(): void
+    {
+        config()->set('atomy.ai.operations.alert_recipient_emails', ['ops@example.test']);
+        config()->set('atomy.ai.operations.alert_cooldown_seconds', 300);
+        config()->set('atomy.ai.operations.log_channel', 'stack');
+        config()->set('atomy.ai.provider.key', 'openrouter');
+
+        $cache = $this->app->make(CacheFactory::class)->store();
+        $clock = $this->createMock(ClockInterface::class);
+        $clock->method('now')->willReturn(new \DateTimeImmutable('2026-04-24T03:00:00+00:00'));
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '29', column '47'). (undefined)

(MissingImport)